packages/app/models/order/fetcher.ts

```diff
@@ -1,5 +1,23 @@
 import { GraphQLClient } from "graphql-request";
-import { getUserOrders, getSubgraphEndpoint, ChainId } from "@stackly/sdk";
+import {
+  getUserOrders,
+  getSubgraphEndpoint,
+  getUserActiveOrders,
+  getUserCancelledOrders,
+  getUserCompleteOrders,
+  ChainId,
+} from "@stackly/sdk";
+
+interface GetOrderBase {
+  chainId: ChainId;
+  address: string;
+  skip?: number;
+  first?: number;
+}
+
+interface GetOrder extends GetOrderBase {
+  currentTimestamp: number;
+}
```

Comment:
This can be joined into 1 and `currentTimestamp` being optional, right?

Reply:
if just make it optional we get type error that might be undefined

<img width="797" alt="image" src="https://github.com/SwaprHQ/stackly-ui/assets/5664434/0ba3edd3-948b-4366-92e6-f8c648517e46">

Reply:
Let's skip this